index.js

```diff
@@ -64,7 +64,7 @@ axeTestUrls(urls, program, {
 	 */
 	onTestComplete: function logResults(results) {
     if (program.jsonDump) {
-      console.log(results)
+      console.log(JSON.stringify(results))
```

Comment:
@WilcoFiers I've run into a weird bug here. For most pages this works fine, if you check out my fork you can get a JSON output like this:

```
./axe-cli -j http://example.com
```
But for some pages I get a `Converting circular structure to JSON` error when trying to convert the results to JSON, for example:

```
$ ./axe-cli -j http://validationhell.com
Error: "Converting circular structure to JSON"
 $s TypeError: Converting circular structure to JSON
    at JSON.stringify (<anonymous>)
    at Object.logResults [as onTestComplete] (/Users/jaime/code/axe-cli/index.js:68:24)
    at /Users/jaime/code/axe-cli/lib/axe-test-urls.js:66:13
    at ManagedPromise.invokeCallback_ (/Users/jaime/code/axe-cli/node_modules/selenium-webdriver/lib/promise.js:1384:14)
    at TaskQueue.execute_ (/Users/jaime/code/axe-cli/node_modules/selenium-webdriver/lib/promise.js:3092:14)
    at TaskQueue.executeNext_ (/Users/jaime/code/axe-cli/node_modules/selenium-webdriver/lib/promise.js:3075:27)
    at asyncRun (/Users/jaime/code/axe-cli/node_modules/selenium-webdriver/lib/promise.js:2935:27)
    at /Users/jaime/code/axe-cli/node_modules/selenium-webdriver/lib/promise.js:676:7
    at process._tickCallback (internal/process/next_tick.js:103:7)
^C
```

The script hangs there and I have to kill it with control+C.

Does it ring a bell? Is there a bug in the results structure? Should we use something like [json-circular](https://www.npmjs.com/package/json-circular) instead?